bin/get_data.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs')
+const process = require('process')
+const TOML = require('@iarna/toml')
+
+const header = 'function addEventListener() {}; ';
+const footer = ' exports.experimentPages = experimentPages; ';
+const body = fs.readFileSync('./workers/redirector.js');
+var experimentPages = eval(header + body + footer);
+
+const stage_domain = "https://www.allizom.org"
+const prod_domain = "https://www.mozilla.org"
+var stage_routes = []
+var prod_routes = []
+
+const toml_config_file = fs.readFileSync('./wrangler.toml');
+const toml_obj = TOML.parse(toml_config_file)
+
+// Validate the data
+function pageValidate(page) {
```

Comment:
What you have here works and I don't suggest needing to change it now, but it feels like this should be a unit test that runs as part of the mocha suite, as opposed to being inside a post-install build script.

Reply:
Yeah, I agree.  I'll take a swipe at doing this today, but may give up if it proves to be too crazy.  First node.js I've ever written :)